libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -40,6 +40,19 @@ void ASTJsonConverter::addJsonNode(
 	bool _hasChildren = false
 )
 {
+  ASTJsonConverter::addJsonNode( _node,
```

Comment:
We have a coding style that might be considered weird by some ;-)

1. Please only use tabs for indentation
2. The rule for breaking "lists of items" (in an abstract sense, applies to anything that has a start token, elements, separators and end token) is as follows: 
  Line break after "opening bracket/start token", each element on its own line with one more level of indentation than the start line, separators at the end of the line and "closing bracket / end token" on its own line at the indentation level of the first line.
In this case, it would look like:
```
{
    ASTJsonConverter::addJsonNode(
        _node,
        _nodeName,
        std::vector<pair<string const, Json::Value const>>(_attributes),
        _hasChildren
    );
}
```

```